scripts/administrative_borders.py

```diff
@@ -47,11 +45,10 @@ def normalise_admin_borders(path_to_nuts, path_to_gadm, path_to_lau, crs, scope_
 
 
 if __name__ == "__main__":
+    shape_dirs = {k: v for k, v in snakemake.input.items() if k != "src"}
```

Comment:
This seems brittle. Isn't it possible to give the whole thing a name?

So instead of `{"nuts": something_with_nuts}` this whole thing would be `{"shapes": {"nuts": something_with_nuts}}`

Reply:
I'm not sure that snakemake can track inputs that are defined inside a nested dictionary...